src/libraries/Common/src/Extensions/Logging/DebuggerDisplayFormatting.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Logging
+{
+    internal static class DebuggerDisplayFormatting
+    {
+        internal static string DebuggerToString(string name, ILogger logger)
+        {
+            ReadOnlySpan<LogLevel> logLevels = stackalloc LogLevel[]
+            {
+                LogLevel.Critical,
+                LogLevel.Error,
+                LogLevel.Warning,
+                LogLevel.Information,
+                LogLevel.Debug,
+                LogLevel.Trace,
+            };
+
+            LogLevel minimumLevel = LogLevel.None;
+
+            // Check log level from highest to lowest. Report the lowest log level.
+            foreach (LogLevel logLevel in logLevels)
+            {
+                if (!logger.IsEnabled(logLevel))
+                {
+                    break;
+                }
+
+                minimumLevel = logLevel;
+            }
+
+            var enabled = minimumLevel != LogLevel.None;
+            var debugText = $@"Name = ""{name}"", Enabled = {(enabled ? "true" : "false")}";
+            if (enabled)
+            {
+                debugText += $", MinLevel = {minimumLevel}";
+            }
```

Comment:
Do we need to show both Enabled and MinLevel?  Seems like you only ever need one of them: either always just show MinLevel, or don't show Enabled if MinLevel > LogLevel.None.

Reply:
I think just showing MinLevel by itself would be confusing. Any value we put on it would make it look like there is no min level, rather than nothing is logged. e.g. MinLevel = None, MinLevel = Disabled, etc.

I updated it so "Enabled = true" isn't shown.